R/gh_whoami.R

```diff
@@ -0,0 +1,74 @@
+#' Info on current GitHub user and token
+#'
+#' Reports wallet name, GitHub login, and GitHub URL for the current
+#' authenticated user, the first few and last characters of the token, and the
+#' associated scopes.
```

Comment:
Hmmm. I am not a security expert, but I would not print out anything from the token. I know that the user has access to it, but I just don't want it to appear in printouts, Rmds, etc.

Let me think about this a bit. I see that it can be useful for beginners, but I am still concerned.